bugbot/rules/webcompat_score.py

```diff
@@ -0,0 +1,88 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import Any, Optional
+
+from bugbot import gcp
+from bugbot.bzcleaner import BzCleaner
+
+
+class WebcompatScore(BzCleaner):
+    def __init__(self):
+        super().__init__()
+        self.autofix_changes = {}
+        self.scored_bugs = {}
+
+    def description(self) -> str:
+        return "Update WebCompat score fields"
+
+    def filter_no_nag_keyword(self) -> bool:
+        return False
+
+    def has_default_products(self) -> bool:
+        return False
+
+    def get_autofix_change(self) -> dict[str, Any]:
+        return self.autofix_changes
```

Comment:
The method on the parent class is doing the same; no need to override it: https://github.com/mozilla/bugbot/blob/8ede0c2b3a2721c9aa6fe2766fa53146261ef47c/bugbot/bzcleaner.py#L575-L577